build.sbt

```diff
@@ -80,7 +83,9 @@ lazy val lambdaCommon = (project in file("lambda/common")).
       "com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",
       "ch.qos.logback" %  "logback-classic" % "1.2.3",
       "com.amazonaws" % "aws-java-sdk-config" % "1.11.246",
-      "com.fasterxml.jackson.core" % "jackson-databind" % "2.8.10"
+      "com.fasterxml.jackson.core" % "jackson-databind" % "2.8.11",
+      "com.fasterxml.jackson.core" % "jackson-annotations" % "2.8.11",
+      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor" % "2.8.10",
```

Comment:
Do we need to have these three twice? (lines 36, 37 and 38 above)

Reply:
Good catch!